src/GeneralStateTestsFiller/stEWASMTests/storedMalformedBytecodeHasStartFiller.yml

```diff
@@ -0,0 +1,146 @@
+# Test malformed bytecode (has start) in init on create
```

Comment:
Could you verify this test case?, for me CREATE is returning 0 (success).

Reply:
Just reconfirmed that it's working for me, this is what I see. Using the `verifycontract` branch of hera (6b26c07fce527dc9a533fb284a284e0025333dfc).

```
> aleth/build/test/testeth -t GeneralStateTests/stEWASMTests -- --testpath ~/eth/tests/ --singlenet Byzantium --jsontrace '' --vm ~/eth/hera3/build/src/libhera.dylib --singletest storedMalformedBytecodeHasStart
[2018-10-18 11:59:02.737715] [0x00007fffac722380] [info]    Loaded EVMC module: hera 0.1.0-dev+commit.6b26c07f (~/eth/hera3/build/src/libhera.dylib)
Running tests using path: "~/eth/tests/"
Running 1 test case...
Test Case "stEWASMTests":
100%
Executing message in Hera
importGlobals
create 80 100 ac 32
Executing message in Hera
importGlobals
finish 0 53
Contract contains start function.
storageStore 0 20
storageStore 40 60
{"stateRoot": "783ef872ba879c8a4a3c60abd90a65839d50913a44c2780f8c0e5df94b7b4f2f"}
*** No errors detected
```